mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -14,6 +15,8 @@ import (
 	containerBeta "google.golang.org/api/container/v1beta1"
 )
 
+var clusterIdRegex = regexp.MustCompile("projects/(?P<project>[^/]+)/locations/(?P<location>[^/]+)/clusters/(?P<name>[^/]+)")
```

Comment:
Would we want to implement this in a shared way?

maybe the inverse of https://github.com/hashicorp/terraform-provider-google/blob/master/google/resource_cloud_asset_folder_feed.go#L218

Reply:
hm, this is literally [copy-pasted from the cluster import id parsing code](https://github.com/hashicorp/terraform-provider-google/blob/170d973d36b36e55a8e4661116cf5b1d423e3acc/google/resource_container_cluster.go#L3289), which _is_ handwritten, so we could reuse it relatively easily.

On the one hand, reusing a variable would mean the two wouldn't be able to get out of sync; on the other hand, if they did get out of sync, tests would likely start failing anyway.

I don't know that there are any other cases where parseImportId is called with a variable, so I would be a little concerned about making that code seem more inaccessible by making it appear more different than other resources. My preference for now would be to just use this here; we should be able to easily rename / move it later if needed.

Reply:
sure that sounds good to me